include/micm/process/cuda_process_set.cuh

```diff
@@ -1,40 +1,17 @@
 #pragma once
```

Comment:
Please add the copyright and license text to the top of all the files in `src/` and `include/`

Reply:
Done.